pontoon/test/fixtures/base.py

```diff
@@ -19,6 +20,17 @@ def client_superuser(client, admin):
     return client
 
 
+@pytest.fixture
+def system_user():
+    """Add the system user to the test data."""
+    return factories.UserFactory(email="pontoon-sync@example.com")
+
+
+@pytest.fixture
+def pretranslation_user():
```

Comment:
Nit: `tm_user`.

Reply:
As mentioned, we should also use this user in https://github.com/mozilla/pontoon/blob/main/pontoon/pretranslation/tests/test_pretranslate.py.

Please also move the `gt_user` here.